Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -575,6 +575,15 @@ public extension Tensor {
     }
 }
 
+public extension Tensor where Scalar: Numeric {
+    /// Returns a `Tensor` by clipping tensor values to a specified min and max.
+    @inlinable
+    @differentiable(wrt: self where Scalar: TensorFlowFloatingPoint)
+    func clipped(min minimum: Tensor, max maximum: Tensor) -> Tensor{
```

Comment:
1. Just using `min` and `max` as variable names is fine.
2. Add a space before `{`.
```suggestion
    func clipped(min: Tensor, max: Tensor) -> Tensor {
```